src/panel/NoResultPanel.jsx

```diff
@@ -1,26 +1,18 @@
 /* globals _ */
 import React from 'react';
 import { Panel } from 'src/components/ui';
+import NoResultMessage from './NoResultMessage';
 
 const close = () => window.app.navigateTo('/');
 
-const NoResultPanel = ({ resetInput }) => {
-  const tryNewSearch = e => {
-    e.preventDefault();
-    close();
-    resetInput();
-  };
-
+const NoResultPanel = () => {
   return (
     <Panel close={close}>
       <div style={{ padding: '16px 16px 32px 16px' }}>
-        <p className="u-mb-xs u-text--smallTitle">{_('No results found.')}</p>
-        <p className="u-text--subtitle u-mb-l">
-          {_('Check the spelling of your search or add more details, such as city or country.')}
+        <NoResultMessage />
+        <p className="u-center">
```

Comment:
As said before on Mattermost, let's not include that link in this PR, as we don't even know when it will be available in prod.
It will be very easy to add later.